api/tests.py

```diff
@@ -9,4 +15,31 @@ def setUp(self):
         pass
 
     def test_example(self):
-        self.assertTrue(True)
\ No newline at end of file
+        self.assertTrue(True) Unit Tests: https://docs.djangoproject.com/en/2.0/topics/testing/overview/
+# Django, Automated Unit Testing Tutorial: https://docs.djangoproject.com/en/2.0/intro/tutorial05/
+
+class ExampleTestCase(TestCase):
+    def setUp(self):
+        pass
+
+    def test_example(self):
+        self.assertTrue(True)
+
+class ExampleModelTestCase(TestCase):
+    def setUp(self):
+        pass
+
+    def test_model(self):
+        self.assertTrue(True)
+
+# https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.SimpleTestCase
+class ExampleSimpleTestCase(SimpleTestCase):
+    def test_simple_example(self):
+        self.assertTrue(True)
+
+# https://docs.djangoproject.com/en/2.0/topics/testing/tools/#transactiontestcase
+class ExampleTransactionTestCase(TransactionTestCase):
+    def test_transaction_example(self):
+        self.assertTrue(True)
+
+# Create your tests here.
```

Comment:
These are my dummy unit test cases which I put in a long time ago, which we want to get rid of. I have replaced them with working examples that test each endpoint, in my disaster-resilience repo (including your sandbox endpoints). 

Brian and Mike have wanted the dummy examples replaced with working ones for some time, so now would be a good time to update them in the backend-exemplar and your new repo. 

https://github.com/hackoregon/disaster-resilience-backend/blob/development/api/tests/tests.py

I can submit the changes from my repo to this PR, or merge them into the backend-exemplar directly, based on what we decide to do here.

Reply:
I am personally completely fine with it if this PR simply serves as a place for us to have this conversation and look at the differences holistically and then if changes are implemented through more specific and narrow PRs. My sense is that would probably be the best way to go but I would defer to others especially @MikeTheCanuck

Reply:
I am going to create a separate PR to merge my api/tests.py changes into the backend-exemplar's staging branch. At that point, this PR should pick up those changes and the diffs will change.